patch_vm.yml

```diff
@@ -37,19 +47,31 @@
       register: vm_facts
       delegate_to: "{{ item.vm_name }}"
 
-    - name: Take a VM Snapshot of VM {{ item.vm_name }}
+    - name: Take a VM Snapshot of qemu VM {{ item.vm_name }}
       uri:
         method: POST
         validate_certs: no
-        url: "https://{{ proxmox_api_host }}:{{ proxmox_api_port }}/api2/json/nodes/{{ vm_node }}/qemu/{{ vm_id }}/snapshot"
+        url: "https://{{ proxmox_api_host }}:{{ proxmox_api_port }}/api2/json/nodes/{{ vm_node }}/{{ vm_type }}/{{ vm_id }}/snapshot"
         headers: "{{ proxmox_api_cookie }}"
         body_format: form-urlencoded
         body:
           snapname: "snap_before_patch-{{ ansible_date_time.date }}"
           description: "Snapshot taken by Update Automation"
           vmstate: 1
```

Comment:
vmstate is not supported for lxc, thus the dupplication for lxc. How to make such an attribut conditional?